cardano-tracer/cardano-tracer.cabal

```diff
@@ -280,6 +280,9 @@ test-suite cardano-tracer-test
   type:                 exitcode-stdio-1.0
   default-extensions:   OverloadedStrings
 
+  if os(windows)
+    buildable:          False
```

Comment:
Are you sure it's the right thing to exclude Windows? Is it possible to trace without `cardano-tracer`?  This is imporant for full node wallets.